modular_skyrat/master_files/code/modules/pain_shock/status_indicator.dm

```diff
@@ -0,0 +1,198 @@
+#define PLANE_STATUS -12 //Status Indicators that show over mobs' heads when certain things like stuns affect them.
+
+#define VIS_STATUS			24
+
+#define VIS_COUNT			24 //Must be highest number from above.
```

Comment:
What do these stand for? Some documentation is really handy in stuff involving rendering.
```suggestion
#define VIS_STATUS 24
#define VIS_COUNT 24 //Must be highest number from above.
```